src/Http/Header/HeaderField.php

```diff
@@ -33,7 +33,7 @@ interface HeaderField
     /** @var string */
     public const CONTENT_LENGTH = 'Content-Length';
 
-    /** @var string  */
+    /** @var string */
```

Comment:
ich behaupte mal ganz frech, dass die begrenzten Typisierungs-Möglichkeiten von Konstanten dazu führen, dass wir diese `@var`-Annotationen auch einfach wegschmeißen können 🙂 (nicht, dass das in diesem PR erfolgen müsste)